docs/source/changelog.md

```diff
@@ -2,6 +2,49 @@
 
 <!-- PR link template: - [#](https://github.com/jupyterhub/kubespawner/pull/) ([@](https://github.com/)) -->
 
+### [1.0]
+
+### [1.0.0]
+
+This release is the continuation of version 0.16.1 and could have been 0.17.0 in
+practice. We opted to release 1.0.0 as it enables us to communicate changes
+according to [SemVer](https://semver.org/). Using SemVer versioning, a change in
+each of the three version numbers (major.minor.patch) represents a different
+kind of change.
+
+### Breaking changes
+
+- When using KubeSpawner 1.0.0 or later together with JupyterHub 1.4.1 or later,
+  deleting a user or removing a named server will imply removal of the
```

Comment:
```suggestion
  deleting a user or removing a named server will remove the
```
unless I've misunderstood and you're trying to communicate some other behaviour?

Reply:
Ah :) I just pushed a change updating that language, what do you think of it now?

> - When using KubeSpawner 1.0.0 or later together with JupyterHub 1.4.1 or later,
  deleting a JupyterHub user or deleting (not just stopping) a named server will
  lead to removing the associated PVC resource. To opt out of this behavior set
  the `delete_pvc` configuration to `False`.

Reply:
LGTM